File: CS/Dashboard_NumericGroupInterval/Form1.cs
using DevExpress.XtraBars.Ribbon;

namespace Dashboard_NumericGroupInterval {
    public partial class Form1 : RibbonForm {
        public Form1() {
            InitializeComponent();
            dashboardDesigner1.CreateRibbon();
            Size = new System.Drawing.Size(1200, 600);
            dashboardDesigner1.LoadDashboard(@"..\..\Data\NumericGroupIntervalFunctionWithMathValues.xml");
        }
    }
}

File: VB/Dashboard_NumericGroupInterval/Form1.vb
Imports DevExpress.XtraBars.Ribbon

Namespace Dashboard_NumericGroupInterval
	Partial Public Class Form1
		Inherits RibbonForm

		Public Sub New()
			InitializeComponent()
			dashboardDesigner1.CreateRibbon()
			Size = New System.Drawing.Size(1200, 600)
			dashboardDesigner1.LoadDashboard("..\..\Data\NumericGroupIntervalFunctionWithMathValues.xml")
		End Sub
	End Class
End Namespace
